app/src/main/res/values/strings.xml

```diff
@@ -704,24 +704,24 @@
         <item quantity="other">%d selected</item>
     </plurals>
     <string name="feed_group_dialog_empty_name">Empty group name</string>
-    <string name="feed_group_dialog_delete_message">Do you want to delete this group?</string>
+    <string name="feed_group_dialog_delete_message">Delete this group?</string>
```

Comment:
I'd keep all of these "Do you want to" everywhere

Reply:
Why?